rpm-testing/Vagrantfile

```diff
@@ -22,118 +16,21 @@ Vagrant.configure(2) do |config|
   config.vm.network "public_network"
 
   # VirtualBox Guest Additions not installed in "centos/7" box, using rsync for now!
-  config.vm.synced_folder "./", "/am-packbuild/rpm-testing", type: "rsync", rsync__enclude: ".git/"
-
-  # Make the local repository available.
-  if repofile == 'archivematica-local.repo'
-    config.vm.synced_folder "../rpm/_yum_repository", "/am-packbuild/_yum_repository", type: "rsync", rsync__enclude: ".git/"
-  end
+  config.vm.synced_folder "../", "/am-packbuild", type: "rsync", rsync__enclude: ".git/"
```

Comment:
We added a way to troubleshoot guest additions in deploy-pub ( https://github.com/artefactual/deploy-pub/commit/cadb91f407fbb4fdc9bba56775a6881452d54634 ) that should be applied here too.

This way, we can share the folder, instead of a one-time, one-way rsync

Reply:
Ok but that's definitely not a blocker so I've a new bug: https://github.com/artefactual-labs/am-packbuild/issues/132.